packages/bib-generator/1.0.0/_manifest.yml

```diff
@@ -0,0 +1,6 @@
+name: "bib-generator"
+title: "Bib Generator"
+description: Generate a BibTex entry by searching through DBLP
+version: 1.0.0
+author: Xeophon
+tags: ["bib", "academia", "research"]

```

Comment:
It may be helpful to add a `homepage:` tag, even if it just links to https://github.com/Xeophon/hub/tree/main. We've had problems with the Espanso Hub software when it's missing.